torch/csrc/lazy/backend/lowering_context.h

```diff
@@ -88,6 +88,8 @@ class TORCH_API LoweringContext {
                             const Shape& shape,
                             const std::string& name) = 0;
 
+  virtual void Lower(const Node*) = 0;
```

Comment:
this will require a xla side change.

Reply:
Does this API really needs to be in `lowering_context.h`? It is not being used outside of `lowering_context` constructor, maybe we should just let backend to define this function instead of putting it in the base class?

Reply:
Thats a good point. I guess it doesn't need to be in the top level class.

Reply:
although, it seems to be passing all the XLA tests. Unless you are referring to a change in a PR that is currently in progress

Reply:
I just merge https://github.com/pytorch/xla/commit/185dd3cd45e4a1b9dcbbf88a3c604ce8c5e752f3, which I think will be broken by this change. It is not a hard fix but I felt like maybe we don't need this class in the top level.